app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t(".error_info") %> <%= pluralize(@user.errors.count, t(".error")) %>.
```

Comment:
sử dụng view số nhiều số ít bằng I18n em ơi
https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0